src/tape/ensemble.py

```diff
@@ -472,6 +472,58 @@ def persist(self, **kwargs):
         self.update_frame(self.object.persist(**kwargs))
         self.update_frame(self.source.persist(**kwargs))
 
+    def sample_objects(self, **kwargs):
+        """Selects a random sample of objects.
```

Comment:
Personally, I think it's worth adding a note that we sample each partition but up to you

Reply:
Will do, the docstring is definitely a bit sparse on information at the moment. On a related note, do you think it would be better to ditch the **kwargs and just copy the available kwargs and their docstring explanations directly into the function?

Reply:
I think that would be a good call and add readability for the users, especially since this is an Ensemble-level function